README.md

```diff
@@ -9,50 +9,46 @@ The _nodezoo-npm_ micro-service provides an interface to the
 [NPM][] registry as part of the [NodeZoo][] system.
 
 
-# Install
-1. Clone this repo into a root _/nodezoo_ folder.
-2. Run `npm install`
+If you're using this microservice, and need help, you can:
 
+- Post a [github issue][],
+- Tweet to [@nodezoo][],
+- Ask on the [Gitter][gitter-url].
 
-# Starting
-To start simply run,
 
-```
-npm run start OR npm run start:dev
-```
-
-### Tagging and Logs
-To tag or set up logs for your service pass the relevant switches on start,
+## Running
+This micro-service can be ran as part of the [Nodezoo][] system. Please follow the
+link below for details on obtaining and running the complete system.
 
-```
-npm run start -- --seneca.options.tag=npm --seneca.log.all
-```
+- [Nodezoo: The complete system][System]
 
-## Inbound Messages
-This micro-service recognizes the following messages:
+## Patterns Handled
 
-   * _role:npm,cmd:get_ - get module details by name
-   * _role:npm,cmd:query_ - query module details from NPM
-   * _role:npm,cmd:extract_ - extract relevant data from NPM result
+### `role:npm,cmd:get`
+Get module details by name
 
+```js
+seneca.act(`role:npm,cmd:get`, name:'seneca')
+```
 
-### Outbound Messages
-This micro-service issues the following message:
+### `role:npm,cmd:query`
+Query module details from NPM
 
-   * _role:search,cmd:insert_ - insert module details into search engine index, OPTIONAL
+```js
+seneca.act(`role:npm,cmd:query`, name:'seneca' )
```

Comment:
Need brackets around name:seneca like {name: 'seneca'} same for other patterns missing them.